ts/store/actions/persistedPreferences.ts

```diff
@@ -0,0 +1,9 @@
+import { ActionType, createStandardAction } from "typesafe-actions";
+
+export const preferenceFingerprintIsEnabledSave = createStandardAction(
+  "PREFERENCES_FINGERPRINT_SAVE_SUCCESS"
+)<boolean>();
```

Comment:
what's the boolean for?

Reply:
It's because, in the preference pane, the usage of fingerprint is explicitly saved passing the value of the switch representing the user's choice (a boolean)

Reply:
can you pls add a comment describing the meaning of the parameter? thx

Reply:
or else, instead of a simple boolean, you could have `{ isFinderprintEnabled: boolean }` that is much more obvious in meaning

Reply:
> or else, instead of a simple boolean, you could have `{ isFingerprintEnabled: boolean }` that is much more obvious in meaning

I'll change it as this second suggestion.